.github/workflows/start-ga.yaml

```diff
@@ -0,0 +1,48 @@
+# Starts a new GA from the latest beta or
+# from another branch, the release version is taken from the fixVersion of the
+# Release ticket supplied as input.
+name: Start new GA release
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseTicket:
+        description: 'releaseTicket (required): it must have a fixVersion which will be used as release version.'
+        required: true
+      mergeBranch:
+        description: 'mergeBranch (optional, default="beta-releases"): the branch to merge from, useful to perform quick fixes or to skip beta.'
+        default: 'beta-releases'
+        required: false
+
+jobs:
+  startRelease:
+    name: Start new GA release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup git
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ^16.x
+          cache: 'npm'
+
+      - name: Install npm@8.19.4
+        run: |
+          npm install -g npm@8.19.4
+
+      - name: Install Dependencies
+        run: |
+          npm -v
+          npm ci
+
+      - name: Start Release
+        env:
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+        run: |
+          node scripts/release.js ga \
```

Comment:
Would it work to use typescript/ts-node to run this file? Any reason we don't want to? Then we could have the file typescript.

Reply:
Sounds simple enough to try, i'm not sure `scripts` is setup for that, i'll check

Reply:
yeah, scripts needs to be restructured and setup for that .. i'm not particularly keen in doing it in this PR, i'd rather do it in a separate PR, and then rebase